kanopy/parser.js

```diff
@@ -22,7 +22,11 @@ module.exports = new Parser(function analyseEC(parsedUrl, ec) {
 
   let match;
 
-  if ((match = /^\/wayf\/product\/(.*)$/i.exec(path)) !== null) {
+  if (/^\/ksa\/.*$/i.test(path)) {
```

Comment:
I would suggest testing for /ksa/tr.

Reply:
Also is ksa_page_title= something worth parsing for?

Reply:
That's the title of the homepage, so I dunno but I'll add it.  :)